core/ssl.c

```diff
@@ -30,7 +30,18 @@ void uwsgi_ssl_info_cb(SSL const *ssl, int where, int ret) {
 }
 
 int uwsgi_ssl_verify_callback(int ok, X509_STORE_CTX * x509_store) {
-        return 1;
+        char buf[256];
+        X509 *err_cert;
+        int depth;
+        int err;
+        depth = X509_STORE_CTX_get_error_depth(x509_store);
+        err_cert = X509_STORE_CTX_get_current_cert(x509_store);
+        X509_NAME_oneline(X509_get_subject_name(err_cert), buf, 256);
+        err = X509_STORE_CTX_get_error(x509_store);
+        if (!ok) {
```

Comment:
if mean

if (!ok) {
509 *err_cert;
+        int depth;
+        int err;
+        depth = X509_STORE_CTX_get_error_depth(x509_store);
+        err_cert = X509_STORE_CTX_get_current_cert(x509_store);
+        X509_NAME_oneline(X509_get_subject_name(err_cert), buf, 256);
+        err = X509_STORE_CTX_get_error(x509_store);
uwsgi_log("....
}


regarding logging i would probably add a check for uwsgi.ssl_verbose like we do in https router